axelrod/strategies/titfortat.py

```diff
@@ -665,3 +665,36 @@ def strategy(self, opponent: Player) -> Action:
         if opponent.history[-1] == D:
             return D
         return C
+
+@FinalTransformer((D,), name_prefix=None)
+class EugineNier(Player):
+    """
+    Plays similar to Tit-for-Tat, but with two conditions:
+    1) Always Defect on Last Move
+    2) If other player defects five times, switch to all defects.
+    
+    Names:
+
+    - EugineNier Strategy: [LessWrong2011]_    
+    """
+
+    name = 'EugineNier'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': False,
+        'makes_use_of': {'length'},
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent: Player) -> Action:
+        if not self.history:
+            return C
+        if opponent.history[-1] == D:
+            return D
+        return C
+        
+        if opponent.history[-5] == [D, D, D, D, D]:
```

Comment:
This line of code is never hit, line 697 will end the function. It is also incorrect: it should be `opponent.history[-5:] == [D, D, D, D, D]`

Move these two lines to just after the initial cooperation and can you please add a test case for this particular instance.